src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -447,6 +447,11 @@ const queryEditorConfiguration: IConfigurationNode = {
 			'description': localize('queryEditor.results.showActionBar', "Whether to show the action bar in the query results view"),
 			'default': true
 		},
+		'queryEditor.results.copyHeaders.delimeter': {
```

Comment:
If we're adding this we should probably also be doing a similar thing for copy results. But that'd be a bigger change - so my question is whether we even need to make this an option right now vs just going with tabs for the time being and then later on coming back and adding functionality to specify the delimiter when copying anything from the results grid. 

Removing this for now would make this change a lot simpler and then we could make sure that we're consistent with our features and how they're handled. 

@kburtram @alanrenmsft Thoughts on this?